src/smart-components/workspaces/WorkspaceListTable.tsx

```diff
@@ -198,6 +254,16 @@ const WorkspaceListTable = () => {
     dispatch(fetchWorkspaces());
   }, [dispatch]);
 
+  useEffect(() => {
+    chrome.getUserPermissions().then((permissions) => {
+      setUserPermissions(
+        permissions.find((obj) => {
+          return obj.permission === 'inventory:groups:write';
+        }),
+      );
+    });
+  }, []);
```

Comment:
Let's add `chrome` to dependency array otherwise on token refresh this wouldn't be calculated
```suggestion
  }, [chrome]);
```

Reply:
Let's try adding `chrome.getUserPermissions`. This funcion is re-initialized when new token is retrieved.

Reply:
@karelhala that worked! Just updated the PR.